ignite/contrib/handlers/tensorboard_logger.py

```diff
@@ -179,12 +179,9 @@ class OutputHandler(BaseOutputHandler):
     Examples:
 
         .. code-block:: python
-
```

Comment:
I think these modifications are wrong, take a look how it is rendered now: https://deploy-preview-2137--pytorch-ignite-preview.netlify.app/generated/ignite.contrib.handlers.tensorboard_logger.html#ignite.contrib.handlers.tensorboard_logger.OutputHandler


Let's also update line 177:
```diff
- Helper handler to log engine's output and/or metrics
+ Helper handler to log engine's output, engine's state attributes and/or metrics
```

Reply:
This line 182 shouldn't be deleted. That's why the docstring is wrongly rendered.